src/component/book/ButtonResponse.tsx

```diff
@@ -0,0 +1,62 @@
+// display list of doctors with DoctorListElement  component
+import React, { useState, Fragment } from "react";
+import axios, { AxiosResponse } from "axios";
+import styled from "styled-components";
+
+const Button = styled.button`
+  padding: 5px 12px;
+  margin-right: 3px;
+  margin-left: 3px;
+  margin-bottom: 2px;
+  color: white;
+  font-size: 12px;
+  font-weight: 700;
+  background-color: #ee737f;
+  border: 0px;
+  border-radius: 3px;
+  appearance: none;
+  cursor: pointer;
+  &:hover {
+    background-color: red;
+  }
+`;
+// response interface after post query
+export interface BookingResponse {
+  id: string;
+  doctorId: string;
+  date: string;
+  createdAt: string;
+  updatedAt: string;
+}
+
+export interface Response {
+  response: string;
+}
+
+const ButtonResponse: React.FC<any> = (params: any) => {
+  const [response, setResponse] = useState<BookingResponse>([] as any);
```

Comment:
`booking` is a better naming here
```suggestion
  const [booking, setBooking] = useState<BookingResponse | null>(null);
```